Common/Data/Model/Course.swift

```diff
@@ -78,6 +78,13 @@ public final class Course: NSManagedObject {
         return self.enrollment != nil && self.enrollment?.objectState != .deleted
     }
 
+    public var openCourseUserActivity: NSUserActivity {
```

Comment:
Please move the code to a separate file/extension (`Course+NSUserActivity.swift`)